openai-assistant/build.properties

```diff
@@ -2,4 +2,5 @@ source.. = src/
 bin.includes = META-INF/,\
                .,\
                plugin.xml,\
-               res/
+               res/,\
+               lib/

```

Comment:
this should solve the problem of not being available at runtime ... you may verify the build produced "openai-assistant.jar" ... whether it contains the 'lib' dir and commonmark

Reply:
Thank @ivy-rew , I tested with 'openai-assistant-10.0.12-SNAPSHOT.jar'. It worked

Reply:
Cool, the `build.properties` is a strange thing coming with the p2-eclipse build universe. You're not the first that stumbled over it :wink: